ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/views/CreateSupportTicketView.kt

```diff
@@ -0,0 +1,328 @@
+            localization = localization,
+        )
+
+        Spacer(modifier = Modifier.height(SECTION_TITLE_BOTTOM_PADDING))
+
+        DescriptionInputField(
+            description = descriptionState.description,
+            onDescriptionChange = descriptionState.onDescriptionChange,
+            enabled = descriptionState.enabled,
+            localization = localization,
+        )
+
+        Spacer(modifier = Modifier.height(SECTION_SPACING))
+
+        SubmitTicketButton(
+            email = emailState.email,
+            description = descriptionState.description,
+            isSubmitting = isSubmitting,
+            onSubmit = onSubmit,
+            localization = localization,
+        )
+    }
+}
+
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_NO or Configuration.UI_MODE_TYPE_NORMAL)
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_YES or Configuration.UI_MODE_TYPE_NORMAL)
+@Composable
+private fun CreateSupportTicketView_Preview() {
+    val mockData = CreateSupportTicketData(
+        onSubmit = { _, _, _, _ -> },
+        onCancel = { },
+        onClose = { },
+    )
+
+    CustomerCenterPreviewTheme {
+        Box(
+            modifier = Modifier
+                .fillMaxSize()
+                .background(MaterialTheme.colorScheme.background),
+        ) {
+            CreateSupportTicketView(
+                mockData,
+                CustomerCenterConfigTestData.customerCenterData().localization,
+            )
+        }
+    }
+}
+
+@Composable
+private fun EmailInputField(
+    state: EmailInputState,
+    localization: CustomerCenterConfigData.Localization,
+) {
+    OutlinedTextField(
+        value = state.email,
+        onValueChange = state.onEmailChange,
+        label = {
+            Text(
+                localization.commonLocalizedString(
+                    CustomerCenterConfigData.Localization.CommonLocalizedString.EMAIL,
+                ),
+            )
+        },
+        placeholder = {
+            Text(
+                localization.commonLocalizedString(
+                    CustomerCenterConfigData.Localization.CommonLocalizedString.ENTER_EMAIL,
+                ),
+            )
+        },
+        isError = state.showError,
+        supportingText = if (state.showError) {
+            {
+                Text(
+                    localization.commonLocalizedString(
+                        CustomerCenterConfigData.Localization.CommonLocalizedString.INVALID_EMAIL_ERROR,
+                    ),
+                )
+            }
+        } else {
+            null
+        },
+        enabled = state.enabled,
+        modifier = Modifier
+            .fillMaxWidth()
+            .onFocusChanged { focusState ->
+                state.onFocusChanged(focusState.isFocused)
+            }
+            .testTag("email_field"),
+        singleLine = true,
+        keyboardOptions = KeyboardOptions(
+            keyboardType = KeyboardType.Email,
+            imeAction = ImeAction.Next,
+        ),
```

Comment:
Does this work? Or does it need the following?

```
keyboardActions = KeyboardActions(onNext = { focusManager.moveFocus(FocusDirection.Down) }),
```

Reply:
I am unsure on this one, I can't tell if Im getting the accurate flow with just the emulator - I'll change to your suggestion and if you have a device I would appreciate a check